woocommerce/wc-admin
# Downloads Report: add summary number, chart and table

Fixes #914.
Part of #915.

Add Summary Number, Chart and Table to _Downloads_ report.

Under the _Product Title_ and _User Name_ columns, we are displaying the ID for now instead of the title/name (see discussion in #915).

Data is loaded from SwaggerHub.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/50144403-752e1f00-02af-11e9-91ac-798b762433a5.png)

### Detailed test instructions:
- Go to the _Downloads_ report.
- Verify the summary number, chart and table are displayed and show data (from SwaggerHub for now).
